### ISSUE TITLE
Encoding problem

### ISSUE THREAD

**Issue by mvnaz** [spotorm/spot2#263]
Hello, I use that library. I have problem with encoding. 

It doesn't show russian characters correctly, and also chars like that " Frühlingsausläufer überall "

I tried:

'charset' => 'utf8mb4'

**Comment by FlipEverything** [comment]
I don't think that this is Spot related. You can set charset like this: #227 Maybe you have to set the charset on the [client side too](https://stackoverflow.com/a/5056667). You may check the database structure as well.

If you need further assistance, please be more specific and post the related code.

**Comment by dertin** [comment]
Maybe someone can add these two methods to the configuration class. How do they see it?

```
  $cfg = new \Spot\Config();

  $cfg->addConnection(DB_TYPE_CONFIG, [
            'dbname' => DB_NAME_CONFIG,
            'user' => DB_USER_CONFIG,
            'password' => DB_PASS_CONFIG,
            'host' => DB_HOST_CONFIG,
            'driver' => DB_DRIVER_CONFIG,
   ]);
  
   $cfg->setCharset('utf8');
   
   // SQL: "SET collation_connection = utf8_czech_ci"
   $cfg->setCollation('utf8_spanish_ci');

```
